### PR TITLE
[Fix] [Translations] [English]: Syntax fix and format tip section

### PR DESCRIPTION
**Pull request by habemuscode**

### CODE REVIEW

File: site/docs/advanced/deployment.md [hunk 1]
```diff
@@ -57,7 +57,10 @@ This can be done with grammY like so:
 2. Define and send sample update objects to your bot via `bot.handleUpdate` ([API reference](https://doc.deno.land/https://deno.land/x/grammy/mod.ts/~/Bot#handleUpdate)).
    Consider to take some inspiration from [these update objects](https://core.telegram.org/bots/webhooks#testing-your-bot-with-updates) provided by the Telegram team.
 
-::: tip Contribute a Testing Framework
+::: Tip
```
Comment (KnorpelSenf) [inline site/docs/advanced/deployment.md:60]: Was this change intentional? Please revert.

https://61b6281f1fcd5ea535276b98--grammy.netlify.app/advanced/deployment.html#testing

Comment (habemuscode) [reply site/docs/advanced/deployment.md]: Done

**Review by KnorpelSenf — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
I didn't notice the target branch is feat/english-transpations. What is the purpose of that branch? When would it be merged into main? Why would we ever want the original docs and the English translation to diverge? (What is even the difference?)
